packages/help-center/src/components/help-center-more-resources.tsx

```diff
@@ -95,6 +97,21 @@ export const HelpCenterMoreResources = () => {
 						</a>
 					</div>
 				</li>
+				<li className="inline-help__resource-item">
+					<div className="inline-help__resource-cell">
+						<a
+							href={ `https://wordpress.com/home/${ window.location.host }?myHomeCoursePaymentsModal=payments-features` }
```

Comment:
I think this might not work for Calypso posts/pages. window.location.host will return wordpress.com for https://wordpress.com/post/bogdanu91atrevert.wordpress.com/2420 for example.

Reply:
We should probably use `getSiteSlug` from `calypso/state/sites/selectors`;